src/renderer/components/settings/module.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div class="module" v-bind:class="{ loading: loading }">
-    <div class="dimmer">
-      <div class="large slow loader"></div>
+  <div class="module">
+    <div class="ui inverted dimmer" :class="{ active: loading }">
```

Comment:
This loader on the entire page jumps for a split second then disapears. You can get a smoother effect by simply switching the text in the page you clicked on with a [nothced circle loading icon](https://semantic-ui.com/elements/icon.html#loading).